content/projects/oop/animals/part2/_index.md

```diff
@@ -156,6 +170,25 @@ Test -> Does cat eat food should Fail
 
 ```
 
+
+### For Python
+
+Use pytest to test your code.
+
+Please refer to the following to find out more: {{% contentlink path="topics/python-specific/automated-testing-with-pytest" %}}
+
+```
+# Python
+
+dog = Dog()
+dog.eats()    # returns 'Food'
+dog.sounds()  # returns 'Bark'
```

Comment:
Since we arent making changes to sounds in part 2 anymore, should we remove its examples ? or its still fine there?

Reply:
So we are making changes to eats(). Since in the first part the method returns "Rax eats" but in the second part there is an update to only return "Food"
So I guess we can maybe remove part of the example that has already been done in part one? i.e sounds()?
What do you think?

Reply:
yeah, i think so too, ill get on it